themes/ht/ht.theme.sh

```diff
@@ -0,0 +1,82 @@
+#! bash oh-my-bash.module
+
+# Harrison's Theme (ht)
+
+# Description:
+# Simple prompt that shows basic working directory and exit code info
+# Works with 8 color terminals
+
+# Example prompt:
+# ● [harrison@ubN2] in src ± |dev ✗|
+
+# OMB SCM Prompt overrides
+SCM_GIT_SHOW_MINIMAL_INFO="true"
+SCM_THEME_PROMPT_DIRTY=" ${_omb_prompt_red}✗${_omb_prompt_normal}"
+SCM_THEME_PROMPT_CLEAN=" ${_omb_prompt_green}✓${_omb_prompt_normal}"
+SCM_NONE_CHAR=""
+
+function _omb_theme_ht_exit_color {
+  case $1 in
+  0)
+    echo "$_omb_prompt_green"
+    ;;
+  1)
+    echo "$_omb_prompt_red"
+    ;;
+  2)
+    echo "$_omb_prompt_gray"
+    ;;
+  126)
+    echo "$_omb_prompt_cyan"
+    ;;
+  127)
+    echo "$_omb_prompt_magenta"
+    ;;
+  130)
+    echo "$_omb_prompt_black"
+    ;;
+  148)
+    echo "$_omb_prompt_yellow"
+    ;;
+  *)
+    echo "$_omb_prompt_blue"
+    ;;
+  esac
+}
+
+# Displays the current prompt
+function _omb_theme_PROMPT_COMMAND {
+  # Capture exit code
+  # NOTE: DO NOT MOVE
+  local EXIT_CODE=$?
+
+  # Start prompt blank
+  PS1=""
+
+  # Exit code indicator
+  PS1+="$(_omb_theme_ht_exit_color "$EXIT_CODE")●$_omb_prompt_reset_color"
+
+  # Environment info
+  PS1+=" $_omb_prompt_red$(_omb_prompt_get_ruby_env)$_omb_prompt_reset_color"
+  PS1+=" $_omb_prompt_green$(_omb_prompt_get_python_env)$_omb_prompt_reset_color"
```

Comment:
@HarrisonGagnon I tried `ht` in my environment, but I noticed that the appearance is slightly different from the presented picture:

> ![ht-dark.png](https://raw.githubusercontent.com/ohmybash/oh-my-bash/fb0c7c2e52cdbfa5df27afdcad6967d6830629c5/themes/ht/ht-dark.png)

In my environment, it looks like

![image](https://github.com/user-attachments/assets/7d09be19-4c2e-4e93-918a-12e6033316ba)

A large space exists between `●` and `[user@host]`.

This is caused by the above parts for the Ruby and Python environments. They add two spaces regardless of whether we are currently in the Ruby or Python environment. However, in the presence of the Ruby or Python environment, `_omb_prompt_get_ruby_env` and `_omb_prompt_get_python_env` are already configured to include a space at the beginning by default:

https://github.com/ohmybash/oh-my-bash/blob/f77615fe938f2809bcd6016bf34ddd793d226664/lib/omb-prompt-base.sh#L61-L90

Reply:
Can I remove the spaces?

```suggestion
  # Environment info
  PS1+="$_omb_prompt_red$(_omb_prompt_get_ruby_env)$_omb_prompt_reset_color"
  PS1+="$_omb_prompt_green$(_omb_prompt_get_python_env)$_omb_prompt_reset_color"
```

Instead, I think I need to add a space to the next `PS1+=`.